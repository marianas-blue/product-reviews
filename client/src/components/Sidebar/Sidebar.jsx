/* eslint-disable no-useless-constructor */
/* eslint-disable react/prefer-stateless-function */
/* eslint-disable import/extensions */
import React from 'react';
import styled from 'styled-components';
import Stars from './Stars/Stars.jsx';
import Feature from './Feature/Feature.jsx';
import Interests from './Interests/Interests.jsx';
import Thoughts from './Thoughts/Thoughts.jsx';

const Boxes = styled.div`
  flex-direction: column;
  align-items: stretch;
  justify-content: stretch;
  padding: 10px;
  padding-right: 50px;
`;

class Reviews extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div>
        <Boxes>
          <Stars />
        </Boxes>
        <Boxes>
          <Feature />
        </Boxes>
        <Boxes>
          <Interests />
        </Boxes>
        <Boxes>
          <Thoughts />
        </Boxes>
      </div>
    );
  }
}

export default Reviews;
