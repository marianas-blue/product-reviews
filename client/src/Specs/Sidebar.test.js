import React from 'react';
import { shallow } from 'enzyme';

import Sidebar from '../components/Sidebar/Sidebar';

describe('Sidebar', () => {
  it('should render correctly', () => {
    
    const component = shallow(<Sidebar/>);

    expect(component).toMatchSnapshot();
  });
});
