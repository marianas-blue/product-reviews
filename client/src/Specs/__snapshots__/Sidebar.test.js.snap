// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar should render correctly 1`] = `
<div>
  <styled.div>
    <Stars />
  </styled.div>
  <styled.div>
    <Feature />
  </styled.div>
  <styled.div>
    <Interests />
  </styled.div>
  <styled.div>
    <Thoughts />
  </styled.div>
</div>
`;
